docs/040_pepr-tutorials/030_create-pepr-operator.md

```diff
@@ -250,15 +231,14 @@ export const RegisterCRD = () => {
     });
 };
 (() => RegisterCRD())();
-
 ```
 
-Finally add a `validate.ts` file to the `crd` folder and add the following:
+Finally add a `validate.ts` file to the `crd` folder and add the following, this will ensure that instances of the WebApp resource are in valid namespaces and have a maximum of 7 replicas.
```

Comment:
```suggestion
Finally add a `validate.ts` file to the `crd` folder and add the following. This will ensure that instances of the WebApp resource are in valid namespaces and have a maximum of 7 replicas.
```